# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/gamepad/GamepadMonitoring.h
# Commit: b9e6371846e0
# Full Hash: b9e6371846e0ae62c4cd5826c182f7b12eec1620
# Author: Chris Martin <cmartin@mozilla.com>
# Date: 2020-08-07 21:36:18
# Regressor Bug: 1657404
# File Overlap Count: 1
# Description:
#   Bug 1657404 - Refactor gamepad monitoring - Part 2 r=daoshengmu
#   
#   Substitute and eliminate MaybeStopGamepadMonitoring(). The logic to do this
#   is so dependent on GamepadPlatformService that it should probably just be a
#   member.
# ==============================================================================

diff -r 550630983bc5 -r b9e6371846e0 dom/gamepad/GamepadMonitoring.h
--- a/dom/gamepad/GamepadMonitoring.h	Fri Aug 07 13:32:17 2020 +0000
+++ b/dom/gamepad/GamepadMonitoring.h	Fri Aug 07 13:27:55 2020 +0000
@@ -9,9 +9,6 @@
 
 namespace mozilla {
 namespace dom {
-// Functions for platform specific gamepad monitoring.
-
-void MaybeStopGamepadMonitoring();
 
 // These two functions are implemented in the platform specific service files
 // (linux/LinuxGamepad.cpp, cocoa/CocoaGamepad.cpp, etc)